### PR TITLE
Updated and modernized

### PR DESCRIPTION
**Pull request by dawnerd**
No idea whats going on with the tests. They refuse to run for me. Im not sure if we could even accurately test the scroll parts.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  ],
+  "description": "Library to handle lazy loading images",
+  "main": "dist/lazyimg.js",
+  "keywords": [
+    "lazy",
+    "load",
+    "performance"
+  ],
+  "ignore": [
+    "**/.*",
```
Comment (jgallen23) [inline bower.json:16]: why are we ignoring everything?

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "node_modules",
+    "bower_components",
+    "test"
+  ],
+  "dependencies": {
+    "jquery": "~1.8.3"
```
Comment (jgallen23) [inline bower.json:22]: you think we should just do `*`?

File: bower.json (hunk 1 above)
Comment (dawnerd) [inline bower.json:16]: Those are hidden files

Comment (jgallen23) [inline bower.json:16]: AHH :+1:

**Pull request merged**